sonar-python-plugin/src/main/java/org/sonar/plugins/python/ExternalIssuesSensor.java

```diff
@@ -33,28 +33,41 @@
 import org.sonar.api.batch.sensor.SensorDescriptor;
 import org.sonar.api.batch.sensor.issue.NewExternalIssue;
 import org.sonar.api.batch.sensor.issue.NewIssueLocation;
+import org.sonar.api.config.Configuration;
 import org.sonar.api.rules.RuleType;
 import org.sonar.api.utils.log.Logger;
+import org.sonar.plugins.python.pylint.PylintSensor;
 import org.sonarsource.analyzer.commons.ExternalReportProvider;
 import org.sonarsource.analyzer.commons.internal.json.simple.parser.ParseException;
 
 public abstract class ExternalIssuesSensor implements Sensor {
 
   private static final int MAX_LOGGED_FILE_NAMES = 20;
   private static final Long DEFAULT_CONSTANT_DEBT_MINUTES = 5L;
+  private static final String PYLINT_LEGACY_KEY = "sonar.python.pylint.reportPath";
 
   @Override
   public void describe(SensorDescriptor descriptor) {
     descriptor
-      .onlyWhenConfiguration(conf -> conf.hasKey(reportPathKey()))
+      .onlyWhenConfiguration(conf -> conf.hasKey(reportPathKey()) || hasPylintLegacyConfig(conf))
       .onlyOnLanguage(Python.KEY)
       .name("Import of " + linterName() + " issues");
   }
 
+  private boolean hasPylintLegacyConfig(Configuration conf) {
+    // Only for PylintSensor
+    return reportPathKey().equals(PylintSensor.REPORT_PATH_KEY) &&
+      conf.hasKey(PYLINT_LEGACY_KEY);
```

Comment:
Alternative solution: rely on polymorphism and define this condition in PylintSensor.